paasta_tools/kubernetes_tools.py

```diff
@@ -1103,7 +1105,10 @@ def get_container_env(self) -> Sequence[V1EnvVar]:
             V1EnvVar(name=name, value=value)
             for name, value in self.get_env().items()
             if name
-            not in list(secret_env_vars.keys()) + list(shared_secret_env_vars.keys())
+            # We drop PAASTA_CLUSTER here and get it instead via `get_kubernetes_environment()`
+            not in list(secret_env_vars.keys())
+            + list(shared_secret_env_vars.keys())
+            + ["PAASTA_CLUSTER"]
```

Comment:
thoughts on filtering out this env var at https://sourcegraph.yelpcorp.com/Yelp/paasta@691c69b75faa114fc654977b9f3401fbe7462d74/-/blob/paasta_tools/kubernetes_tools.py?L1076:5&subtree=true instead of here? it should have the same effect, but in a more centralized location